pygmt/src/shift_origin.py

```diff
@@ -44,23 +65,63 @@ def shift_origin(
 
     Examples
     --------
+
+    Shifting the plot origin permanently:
+
     >>> import pygmt
     >>> fig = pygmt.Figure()
-    >>> fig.basemap(region=[0, 10, 0, 10], projection="X10c/10c", frame=True)
-    >>> # Shift the plot origin in x direction by 12 cm
-    >>> fig.shift_origin(xshift=12)
-    >>> fig.basemap(region=[0, 10, 0, 10], projection="X14c/10c", frame=True)
-    >>> # Shift the plot origin in x direction based on the previous plot width
-    >>> # Here, the width is 14 cm, and xshift is 16 cm
-    >>> fig.shift_origin(xshift="w+2c")
+    >>> fig.basemap(region=[0, 5, 0, 5], projection="X5c/5c", frame=True)
+    >>> # Shift the plot origin in x direction by 6 cm
+    >>> fig.shift_origin(xshift=6)
+    <contextlib._GeneratorContextManager object at ...>
+    >>> fig.basemap(region=[0, 7, 0, 5], projection="X7c/5c", frame=True)
+    >>> # Shift the plot origin in x direction based on the previous plot width.
+    >>> # Here, the width is 7 cm, and xshift is 8 cm.
+    >>> fig.shift_origin(xshift="w+1c")
+    <contextlib._GeneratorContextManager object at ...>
+    >>> fig.basemap(region=[0, 10, 0, 5], projection="X10c/5c", frame=True)
+    >>> fig.show()
```

Comment:
Hm. I am wondering what the line `<contextlib._GeneratorContextManager object at ...>` is and if it could / should be removed? It's not in the example for the temporary usage.
 
```suggestion
    >>> fig.basemap(region=[0, 5, 0, 5], projection="X5c/5c", frame=True)
    >>> # Shift the plot origin in x direction by 6 cm
    >>> fig.shift_origin(xshift=6)
    >>> fig.basemap(region=[0, 7, 0, 5], projection="X7c/5c", frame=True)
    >>> # Shift the plot origin in x direction based on the previous plot width.
    >>> # Here, the width is 7 cm, and xshift is 8 cm.
    >>> fig.shift_origin(xshift="w+1c")
    >>> fig.basemap(region=[0, 10, 0, 5], projection="X10c/5c", frame=True)
    >>> fig.show()
```

Reply:
it can be removed.